packages/calypso-color-schemes/package.json

```diff
@@ -19,12 +19,17 @@
 	"bugs": {
 		"url": "https://github.com/Automattic/wp-calypso/issues"
 	},
-	"main": "dist/calypso-color-schemes.css",
+	"main": "css/index.css",
 	"publishConfig": {
 		"access": "public"
 	},
 	"scripts": {
 		"clean": "check-npm-client && npx rimraf dist src/__color-studio",
 		"prepare": "check-npm-client && node bin/prepare-sass-assets.js && node bin/build-css.js"
+	},
+	"devDependencies": {
```

Comment:
Traditionally, in our repo `packages` have only `dependencies`, while all `devDependencies` are in the root `package.json`.

But I don't know if that was a conscious decision or a workaround around some npm behaviour that we don't need anymore with yarn. 

Personally I have no problem with having `devDependencies` on individual packages, just pointing it out.

Reply:
I don't mind it either ways. But it seems to me `devDependencies` is more proper since neither of these packages will make it to the output code.